platforms/hyperledger-besu/configuration/reset-network.yaml

```diff
@@ -30,7 +30,7 @@
     # remove this if not needed
     - name: Uninstall flux
       shell: |
-        yes | KUBECONFIG={{ kubernetes.config_file }} {{ bin_install_dir }}/flux uninstall --namespace=flux-{{ network.env.type }}
+        yes | KUBECONFIG={{ kubernetes.config_file }} kubectl delete namespace flux-{{ network.env.type }}
```

Comment:
We used be using the kubernetes ansible module for deletion of resources instead of shell. For reference, https://github.com/hyperledger/bevel/blob/main/platforms/hyperledger-fabric/configuration/roles/delete/peer_pod/tasks/main.yaml